### PR TITLE
Download starthnstrace.ps1 when running collectlogs.ps1

### PR DESCRIPTION
**Pull request by Keith-Mange**
Download starthnstrace.ps1 when running collectlogs.ps1

### CODE REVIEW

File: Kubernetes/windows/debug/collectlogs.ps1 [hunk 1]
```diff
@@ -21,6 +21,7 @@ ipmo $helper
 DownloadFile -Url  "https://raw.githubusercontent.com/$GithubSDNRepository/master/Kubernetes/windows/debug/dumpVfpPolicies.ps1" -Destination $BaseDir\dumpVfpPolicies.ps1
 DownloadFile -Url "https://raw.githubusercontent.com/$GithubSDNRepository/master/Kubernetes/windows/hns.psm1" -Destination $BaseDir\hns.psm1
 DownloadFile -Url "https://raw.githubusercontent.com/$GithubSDNRepository/master/Kubernetes/windows/debug/starthnstrace.cmd" -Destination $BaseDir\starthnstrace.cmd
```
Comment (daschott) [inline Kubernetes/windows/debug/collectlogs.ps1:23]: Do we want to keep [starthnstrace.cmd](https://github.com/microsoft/SDN/blob/master/Kubernetes/windows/debug/starthnstrace.cmd), or replace it?

File: Kubernetes/windows/debug/collectlogs.ps1 [hunk 2]
```diff
@@ -21,6 +21,7 @@ ipmo $helper
...
+DownloadFile -Url "https://raw.githubusercontent.com/$GithubSDNRepository/master/Kubernetes/windows/debug/starthnstrace.ps1" -Destination $BaseDir\starthnstrace.ps1
 DownloadFile -Url "https://raw.githubusercontent.com/$GithubSDNRepository/master/Kubernetes/windows/debug/startpacketcapture.cmd" -Destination $BaseDir\startpacketcapture.cmd
```
Comment (daschott) [inline Kubernetes/windows/debug/collectlogs.ps1:25]: Should we download [startpacketcapture.ps1](https://github.com/microsoft/SDN/blob/master/Kubernetes/windows/debug/startpacketcapture.ps1) as well?

File: Kubernetes/windows/debug/collectlogs.ps1 (hunk 1 above)
Comment (Keith-Mange) [reply Kubernetes/windows/debug/collectlogs.ps1:23]: I'm going to say "yes" for now.

File: Kubernetes/windows/debug/collectlogs.ps1 (hunk 2 above)
Comment (Keith-Mange) [reply Kubernetes/windows/debug/collectlogs.ps1:25]: Great point!

**Pull request merged**